lib/validators/index.js

```diff
@@ -52,9 +52,14 @@ export const type = (value) => {
     throw new Error(`Parameter "type" is not valid - Value: ${value}`);
 };
 
-// TODO; https://github.com/asset-pipe/core/issues/12
+const pathTraversalPattern = /\.\./;
```

Comment:
Does this validator get used for URL paths at all? Or is it just file paths? If url paths then it probably needs to account for url encoded . characters too like `%2E%2E`